routes/api/users.js

```diff
@@ -71,9 +95,9 @@ router.post('/register', async (req, res) => {
 
 router.post('/login', async (req, res) => {
   try {
-    const { email, password } = req.body;
+    const { email, password, isVerified } = req.body;
```

Comment:
```suggestion
    const { email, password } = req.body;
```
To ci nie przyjdzie w body, `isVerified` bo to jest Twoje zadanie żeby sprawdzić to w bazie danych. Kiedy pobierzesz użytkownika z bazy danych wtedy sprawdzasz ten klucz. Nie ufasz użytkownikowi który mówi ci że jest zweryfikowany.